Fixes a discrepancy in UpdatePath readme

SpaceSmithers opened the pull request:
## About The Pull Request
The recent overhaul of UpdatePath's documentation contains a discrepancy. Currently, it mentions that the line `/mob/living/basic/mouse{name="Tom"} : /mob/living/basic/mouse/tom{name = @SKIP, @OLD}` will keep all the old variables except for 'name'. This format is slightly incorrect, and will actually cause the UpdatePaths script to error and not complete its task at all.

By cross-referencing some pre-existing scripts in the folder, as well as actual testing, the correct way to write this would be the following: `/mob/living/basic/mouse{name="Tom"} : /mob/living/basic/mouse/tom{@OLD;name=@SKIP}`

Note the use of a semi-colon instead of the comma, the removal of spaces, and the use of `@OLD` _before_ `@SKIP`. **Each of these cases are important and affect the outcome/success of the script!**

SpaceSmithers committed:
fixes improper documentation

ShizCalev commented:
Could you document @SUBTYPES while at it perhaps since it's literally not mentioned anywhere lol

SpaceSmithers commented:
> Could you document @SUBTYPES while at it perhaps since it's literally not mentioned anywhere lol

Sure thing.

SpaceSmithers committed:
Minor consistency and clarity change

Edits the paragraph below to explain the actions in the order of execution, as well as mentioning the importance of the order of operations.

SpaceSmithers committed:
oopsies

accidentally duped a paragraph.
